syntax = "proto3";

package langchain;

service LangChain {
  rpc AnswerQuery (UserQuery) returns (QueryAnswer);

}

message UserQuery {
  string query_text = 1;  
}

message QueryAnswer {
  string answer_text = 1;
}


